Makefile

```diff
@@ -99,7 +101,11 @@ python_build : $(PYTHON_SETUP_FILE)
 
 # install python requirements
```

Comment:
I think pip should be executed in this way:
- with _make_, execute _pip --user_
- with _make install_, execute _pip --system_